third_party/2.7/dateutil/parser.pyi

```diff
@@ -0,0 +1,37 @@
+from typing import List, Tuple, Optional, Callable, Union, IO, Any
+from datetime import datetime
+
+__all__ = ...  # type: List[str]
+
+
+class parserinfo(object):
+    JUMP = ... # type: List[str]
+    WEEKDAYS = ... # type: List[Tuple[str, str]]
+    MONTHS = ... # type: List[Tuple[str, str]]
+    HMS = ... # type: List[Tuple[str, str, str]]
+    AMPM = ... # type: List[Tuple[str, str]]
+    UTCZONE = ... # type: List[str]
+    PERTAIN = ... # type: List[str]
+    TZOFFSET = ... # type: Dict[str, int]
+
+    def __init__(self, dayfirst: bool=..., yearfirst: bool=...) -> None: ...
+    def jump(self, name: str) -> bool: ...
+    def weekday(self, name: str) -> Union[int, None]: ...
+    def month(self, name: str) -> Union[int, None]: ...
+    def hms(self, name: str) -> Union[int, None]: ...
+    def ampm(self, name: str) -> Union[int, None]: ...
+    def pertain(self, name: str) -> bool: ...
+    def utczone(self, name: str) -> bool: ...
+    def tzoffset(self, name: str) -> Union[int, None]: ...
+    def convertyear(self, year: int) -> int: ...
+    def validate(self, res: datetime) -> bool: ...
+
+class parser(object):
+    def __init__(self, info: Optional['parserinfo']) -> None: ...
+    def parse(self, timestr: str, default: Optional[datetime],
```

Comment:
On Wed, Jul 6, 2016 at 11:10 AM, Guido van Rossum <notifications@github.com>
wrote:

> No need for a new PR. You can just make another local commit (or more) and
> push again.


Done. Two commits, incorporating this discussion. Thanks for the inputs,
Paul.


commit ef576d4f91b8ad3632841c1e89d82fffd859a687
Author: Skip Montanaro <skipm@trdlnk.com>
Date:   Thu Jul 7 09:20:24 2016 -0500

    str -> unicode, per Paul Ganssle

commit 937b6c5db8cf75de8eb67654090b37af7a7e147d
Author: Skip Montanaro <skipm@trdlnk.com>
Date:   Thu Jul 7 09:17:32 2016 -0500

    tweak a couple definitions

Skip